src/python/grpcio_tests/tests/unit/_signal_handling_test.py

```diff
@@ -208,8 +208,12 @@ def testStreamingHandlerWithException(self):
                 self._handler.await_connected_client()
                 client.send_signal(signal.SIGINT)
                 client.wait()
-                print(_read_stream(client_stderr))
-                self.assertEqual(0, client.returncode)
+                client_stderr_output = _read_stream(client_stderr)
+                try:
+                    self.assertEqual(0, client.returncode)
```

Comment:
Nit: Instead of doing a try-except, you can just use the [`msg` kwarg](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual).

Reply:
Nevermind. I see the intention here.